build/tasks/task-serve.js

```diff
@@ -25,8 +26,14 @@ gulp.task('serve', async () => {
     '--port', ports[0],
   ];
 
+  const runnerArgs = [
+    ...args,
+    ...(yargs.argv.browser ? ['--browser', yargs.argv.browser] : []),
+    ...(yargs.argv.mockOs ? ['--mock-os', yargs.argv.mockOs] : []),
```

Comment:
I'd expect subtle differences in the UI to exist between OSes. At the very least, window controls.